WordPressUI/Ghosts/Internal/UIView+InnerGhost.swift

```diff
@@ -65,14 +65,15 @@ private extension UIView {
 
     /// Enumerates all of the receiver's Leaf Views.
     ///
-    func enumerateLeafViews(callback: (UIView) -> ()) {
-        guard !subviews.isEmpty else {
-            callback(self)
-            return
-        }
+    func enumerateGhostableLeafViews(callback: (UIView) -> ()) {
+        guard isGhostable else { return }
```

Comment:
Our convention is as follows:

```
guard CONDITION else {
    return
}
```

[More here!](https://github.com/wordpress-mobile/swift-style-guide#braces)

Reply:
Oops! Fixed.